### ISSUE TITLE
The Geotail CPI and EPIC instruments should be NASA not JAXA.

### ISSUE THREAD

**Issue by aharonroberts** [hpde/NASA#2]
These are US instruments.  I tried to figure out how to do a pull request, but failed.

**Comment by jweygand** [comment]
Hi Aaron, 

Sorry, I'm not clear here. Which metadata do you mean? 
The instruments are in the SMWG. 
See 
https://hpde.io/SMWG/Instrument/Geotail/EPIC.html 
and 
https://hpde.io/SMWG/Instrument/Geotail/CPI.html 

Do you mean so other metadata? 

thanks, 
j

**Comment by rchimiak** [comment]
Aaron must be speaking about the Numerical descriptors  associated with those instruments.
Up until now we have not split up any missions among agencies, but we may be ready to do more of a fine grain distribution , particularly if the agencies already exit as git repositories as in this case.
I am ready to make the change if Aaron agrees

**Comment by jweygand** [comment]
I wondered about that but wanted to be sure. 
Hopefully we can sort the Numerical descriptors now by agency. 
(I know we should do it this way, but I have a feeling that many 
will be confused since most people are not aware of which agency 
produced each instrument/observatory.) 

thanks, 
j

**Comment by aharonroberts** [comment]
Yes, I mean the Numerical descriptors, but it could be the instruments too (less important, since the former is associated with DOIs for now, not the latter).   So I agree to go ahead.

Thanks.

Aaron

PS: Can someone explain the hpde/NASA and Author in the “To” and “cc” fields in the reply?

**Comment by tressahelvey** [comment]
@jweygand Hi James, I'm looking at this old ticket. Do these still need to be moved from JAXA Naming Authority to NASA Naming Authority?  If so, I will make a note for when we begin the process of moving. If you've already done them, I'll check them off. PS, can you identify a list that would need to be changed?? Thank you. Tressa
